.github/workflows/tests.yaml

```diff
@@ -60,10 +60,11 @@ jobs:
       run: |
         sg microk8s 'juju bootstrap microk8s microk8s'
         sg microk8s 'juju add-model coredns microk8s'
+        sg microk8s 'juju add-model coredns-client microk8s'
 
     - name: Run Integration tests
       run: |
-        KUBECONFIG=/home/runner/.kube/config tox -e integration -- --model=coredns
+        KUBECONFIG=/home/runner/.kube/config tox -e integration -- --model=coredns --client-model=coredns-client
```

Comment:
you may want to add 'coredns-client' to the "Collect kube logs" for when things go badly

Reply:
I added the client namespace to the kube logs. I also updated the app labels that are specified